props/saber_BC_buttons.h

```diff
@@ -265,6 +265,10 @@ Turn off blade -        Hold POW and wait until blade is off,
 #undef PROP_TYPE
 #define PROP_TYPE SaberBCButtons
 
+#ifndef CONFIG_STARTUP_DELAY
+#define CONFIG_STARTUP_DELAY 0
```

Comment:
This is not actually how CONFIG_STARTUP_DELAY works, and doing it this way could cause a problem.
If CONFIG_STARTUP_DELAY isn't defined, it means that it will use a default from the arduino-proffieboard plugin, not zero.

Reply:
I see. Well omission is an easy edit if that's the correct move here?